apps/admin-gui-e2e/src/integration/facility-management-perun_admin.spec.js

```diff
@@ -0,0 +1,225 @@
+      sessionStorage.setItem('baPrincipal', '{"name": "facilityManager"}');
+      sessionStorage.setItem('basicUsername', Cypress.env('BA_USERNAME_FACILITY_MANAGER'));
+      sessionStorage.setItem('basicPassword', Cypress.env('BA_PASSWORD_FACILITY_MANAGER'));
+      cy.visit('service-access');
+    }
+  })
+
+  beforeEach(() => {
+    cy.visit('facilities');
+  })
+
+  it('test create facility', () => {
+
+    cy.intercept('**/facilitiesManager/createFacility?**').as('createFacility')
+      .intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
+      .intercept('**/facilitiesManager/getFacilities').as('getFacilities')
+      .get('[data-cy=new-facility-button]')
+      .click()
+      .wait('@getFacilities')
+      .get('[data-cy=facility-name-input]')
+      .type('test-e2e-facility')
+      .get('[data-cy=facility-description-input]')
+      .type('test-e2e-facility-description')
+      .get('[data-cy=create-facility-button]')
+      .click()
+      .wait('@createFacility')
+      .wait('@getEnrichedFacilities')
+      // assert that the facility was created
+      .get('[data-cy=auto-focused-filter]')
+      .type('test-e2e-facility')
+      .get('[data-cy=test-e2e-facility-checkbox]')
+      .should('exist')
+  });
+
+  it('test delete facility', () => {
+
+    cy.intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
+      .intercept('**/facilitiesManager/deleteFacility?**').as('deleteFacility')
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName1}-checkbox]`)
+      .click()
+      .get('[data-cy=delete-facility-button]')
+      .click()
+      .get('[data-cy=delete-button-dialog]')
+      .click()
+      .wait('@deleteFacility')
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName1}-checkbox]`)
+      .should('not.exist')
+  });
+
+  it('test create resource', () => {
+
+    cy.intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
+      .intercept('**/vosManager/getAllVos').as('getAllVos')
+      .intercept('**/facilitiesManager/getAssignedRichResources?**').as('getAssignedResources')
+      .intercept('**/resourcesManager/createResource?**').as('createResource')
+
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName2}]`)
+      .click()
+      .get('[data-cy=resources]')
+      .click()
+      .get('[data-cy=create-resource-button]')
+      .click()
+      .wait('@getAllVos')
+      .get('[data-cy=create-resource-select-vo]')
+      .click()
+      .get('mat-option')
+      .contains(`${dbVoName}`)
+      .click()
+      .get('[data-cy=create-resource-name-input]')
+      .type('test-e2e-resource')
+      .get('[data-cy=create-resource-dialog-button]')
+      .click()
+      .wait('@createResource')
+      .wait('@getAssignedResources')
+      .get('[data-cy=test-e2e-resource-checkbox]')
+      .should('exist')
+  });
```

Comment:
There will be good to move intercept lower before the call that we want to catch. Maybe we can also remove `intercept` and `wait` for `createResource`.
```suggestion
  it('test create resource', () => {

    cy.intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
      .intercept('**/vosManager/getAllVos').as('getAllVos')

      .wait('@getEnrichedFacilities')
      .get(`[data-cy=${dbFacilityName2}]`)
      .click()
      .get('[data-cy=resources]')
      .click()
      .get('[data-cy=create-resource-button]')
      .click()
      .wait('@getAllVos')
      .get('[data-cy=create-resource-select-vo]')
      .click()
      .get('mat-option')
      .contains(`${dbVoName}`)
      .click()
      .get('[data-cy=create-resource-name-input]')
      .type('test-e2e-resource')
      .get('[data-cy=create-resource-dialog-button]')
      .click()
      .intercept('**/facilitiesManager/getAssignedRichResources?**').as('getAssignedResources')
      .wait('@getAssignedResources')
      .get('[data-cy=test-e2e-resource-checkbox]')
      .should('exist')
  });
```